class.jetpack-gutenberg.php

```diff
@@ -165,8 +181,8 @@ private static function prepend_block_prefix( $block_name ) {
 	 *
 	 * @return string The unprefixed extension name.
 	 */
-	private static function remove_extension_prefix( $extension_name ) {
-		if ( wp_startswith( $extension_name, 'jetpack/' ) || wp_startswith( $extension_name, 'jetpack-' ) ) {
+	public static function remove_extension_prefix( $extension_name ) {
+		if ( 0 === strpos( $extension_name, 'jetpack/' ) || 0 === strpos( $extension_name, 'jetpack-' ) ) {
```

Comment:
I'm not sure why you made this change.

Reply:
Neither am I now, but it was something to do with `wp_startswith` not being available in some context. I'll change it back and see if I can reproduce the error